pkg/nsx/client.go

```diff
@@ -63,18 +67,30 @@ func GetClient(cf *config.NSXOperatorConfig) *Client {
 		log.Error(err, "get version error")
 		return nil
 	}
-	err = nsxVersion.Validate(minVersion)
+	err = nsxVersion.Validate()
 	if err != nil {
 		log.Error(err, "validate version error")
 		return nil
 	}
+
 	queryClient := search.NewQueryClient(restConnector(cluster))
 	groupClient := domains.NewGroupsClient(restConnector(cluster))
 	securityClient := domains.NewSecurityPoliciesClient(restConnector(cluster))
 	ruleClient := security_policies.NewRulesClient(restConnector(cluster))
 	nsxChecker := &NSXHealthChecker{
 		cluster: cluster,
 	}
+	nsxVersionChecker := &NSXVersionChecker{
+		cluster:          cluster,
+		versionSupported: false,
+	}
+
+	if nsxVersion.featureSupported(minVersion) {
+		nsxVersionChecker.versionSupported = true
+	} else {
+		log.Info("warning: check nsx version failed", "nsx version", nsxVersion.NodeVersion)
```

Comment:
`check nsx version failed`: could we print current NSX version and the required version